packages/core/src/components/Search/Search.tsx

```diff
@@ -35,6 +35,7 @@ const Search = forwardRef(
       onFocus,
       onBlur,
       onClear,
+      onEnterKey,
```

Comment:
why aren't we just exposing `onKeyDown`?

Reply:
I actually thought about doing it, but maybe we should make this more straightforward for cases like Enter, which is the one the team currently needs. I can't think of another key that someone might want to check for stuff like Search field

Reply:
@shlomitc after reconsideration, went with your suggestion at the end, thanks!